.goreleaser.yml

```diff
@@ -33,16 +38,25 @@ builds:
       - -trimpath
     ldflags:
       - '-s -w -X "main.version={{ .RawVersion }}" -X "main.prerelease={{ if .IsSnapshot }}snapshot.{{ .ShortCommit }}{{ else }}{{ .Prerelease }}{{ end }}"'
-    targets:
-      - freebsd_386
-      - freebsd_amd64
-      - freebsd_arm
-      - linux_386
-      - linux_amd64
-      - linux_arm
-      - openbsd_386
-      - openbsd_amd64
-      - solaris_amd64
+    goarch:
+      - '386'
+      - amd64
+      - arm
+      - arm64
+    goos:
+      - freebsd
+      - openbsd
+      - linux
+      - solaris
+    ignore:
+      - goarch: '386'
+        goos: darwin
+      - goarch: arm
+        goos: openbsd
+      - goarch: arm64
+        goos: openbsd
+      - goarch: arm64
+        goos: freebsd
```

Comment:
I'm unsure whether the above 3 combinations are ignored intentionally, or by accident as they aren't explicitly mentioned here https://github.com/hashicorp/terraform/blob/c0e37a7ea1928918a9efc02290ecd241d396d3a6/scripts/build.sh#L20 but gox doesn't support them:
https://github.com/mitchellh/gox/blob/master/platform.go

Either way there is no point in providing LS builds if there is no corresponding Terraform build, so I opted to ignore these. 🤷🏻‍♂️